R/system.R

```diff
@@ -985,8 +985,15 @@ getDBConnection <- function(type, host = NULL, port = "", databaseName = "", use
     # if the connection is null or the connection is invalid, create a new one.
     if (is.null(conn) || !DBI::dbIsValid(conn)) {
       # When the Amazon Redshift data source is executed on Linux, it's possible that sslCA parameter is defined, for this case get the file path from environment variable.
-      if(Sys.info()["sysname"] == "Linux" && type =="redshift" && sslCA != ""){
-        sslCA <- Sys.getenv("REDSHIFT_SSL_CERT_FILE");
+      if(Sys.info()["sysname"] == "Linux" && sslCA != ""){
+        if (type =="redshift") {
+          sslCA <- Sys.getenv("REDSHIFT_SSL_CERT_FILE")
+          if (sslCA == "") { # if environment variable is not set, fallback to default one.
+            sslCA <- "/etc/ssl/certs/amazon-trust-ca-bundle.crt"
+          }
+        } else if (type == "posgres") {
```

Comment:
posgres or postgres

Reply:
it's postgres. I fixed the typo. thank you!